src/common-services/api-endpoints.js

```diff
@@ -0,0 +1,7 @@
+export const domain_api = 'https://pl-backend-staging.herokuapp.com/api/domain/';
```

Comment:
Set env variables in netlify for staging and production, pick them here and depending on that select a backend base url. 

```
export const prodBackendUrl = '...'
export const stagBackendUrl = '..'
export const apiBase = isProd ? prodBackendUrl : stagBackendUrl
export const domain_api = `${apiBase}/..` 
```

and so on. Easier to maintain/change later